pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -1349,6 +1349,28 @@ void expireMessagesForAllSubscriptions(String topic, long expireTimeInSeconds)
      */
     CompletableFuture<Message<byte[]>> getMessageByIdAsync(String topic, long ledgerId, long entryId);
 
+    /**
+     * Get message id by timestamp.
```

Comment:
```suggestion
     * Get message ID by timestamp.
```

Pls pay attention to capitalization and correct all occurrences, thanks.